src/main/scala/com/monsanto/arch/cloudformation/codegen/ParseDocs.scala

```diff
@@ -0,0 +1,243 @@
+package com.monsanto.arch.cloudformation.codegen
```

Comment:
This seems like a really cool idea. Can you add some sort guide on how to use it to the README under a how to add new types section?